### ISSUE TITLE
Document program detection

### ISSUE THREAD

**Issue by mattwelborn** [MolSSI/QCEngine#118]
The documentation section "Environment Detection" should provide an example of how to see which programs were detected. I think that qcengine.list_available_programs() does this? (Same goes for procedures.)

**Comment by loriab** [comment]
a `qcengine --info` command might be a good idea. print
* where qcengine is running from
* version of qcengine and qcel
* all interfaced programs (for this qcng version)
* all detected programs along with their version and location
* ditto procedures
* default `Config`

**Comment by dgasmith** [comment]
Agreed, that some CLI data would be useful. Perhaps more inline with the other QCA CLI's:

- `QCEngine info` - Show information as listed above.
- `QCEngine run *program* *json_file/json_data*` - Executes the program from input
- `QCEngine run-procedure *procedure* *json*` - Executes the procedure.

**Comment by mattwelborn** [comment]
@dgasmith 0.9.0 or no?

**Comment by dgasmith** [comment]
I say no, but hopefully before the release. Does the proposed CLI seem reasonable?

**Comment by mattwelborn** [comment]
I'm happy with the proposed CLI.
